blocks/library/quote/style.scss

```diff
@@ -2,7 +2,7 @@
 	margin: 0 0 16px;
 	box-shadow: inset 0px 0px 0px 0px $light-gray-500;
 
-	footer {
+	cite {
```

Comment:
We should also probably keep these rules for a few releases as `cite, footer {}`.

Reply:
@mtias makes sense—for a while. I just did that.